app/components/meeting/edit-meeting-modal.js

```diff
@@ -294,7 +295,28 @@ export default class MeetingEditMeetingComponent extends Component {
         saveActivities.push(this.decisionPublicationActivity.save());
       }
 
+      // Check if an annex meeting exists, if so update its planned start
+      const annexMeeting = yield this.store.queryOne('meeting', {
+        filter: {
+          'main-meeting': {
+            ':id:': this.args.meeting.id,
+          }
+        }
+      });
+      if (annexMeeting?.id) {
+        annexMeeting.plannedStart = this.args.meeting.plannedStart;
+        saveActivities.push(annexMeeting.save());
```

Comment:
I asked in the jira ticket.
Another thing we may want to change is the number.
that is also not editable on annex meetings